graylog-storage-elasticsearch7/src/main/java/org/graylog/storage/elasticsearch7/views/ElasticsearchBackend.java

```diff
@@ -255,7 +254,7 @@ public QueryResult doRun(SearchJob job, Query query, ESGeneratedQueryContext que
                     return new SearchRequest()
                             .source(searchTypeQueries.get(searchTypeId))
                             .indices(indices.toArray(new String[0]))
-                            .indicesOptions(IndicesOptions.fromOptions(false, false, true, false));
+                            .indicesOptions(IndicesOptions.fromOptions(true, true, true, false));
```

Comment:
allowNoIndices is now set to true. Is this by purpose? Does it help with the issue which is solved by this PR?